collector/nginx_plus.go

```diff
@@ -323,3 +354,11 @@ func newUpstreamServerMetric(namespace string, metricName string, docString stri
 func newStreamUpstreamServerMetric(namespace string, metricName string, docString string) *prometheus.Desc {
 	return prometheus.NewDesc(prometheus.BuildFQName(namespace, "stream_upstream_server", metricName), docString, []string{"upstream", "server"}, nil)
 }
+
+func newStreamZoneSyncMetric(namespace string, metricName string, docString string) *prometheus.Desc {
+	return prometheus.NewDesc(prometheus.BuildFQName(namespace, "stream_zone_sync_status", metricName), docString, nil, nil)
+}
+
+func newStreamZoneSyncZonesMetric(namespace string, metricName string, docString string) *prometheus.Desc {
+	return prometheus.NewDesc(prometheus.BuildFQName(namespace, "stream_zone_sync", metricName), docString, []string{"server_zone"}, nil)
```

Comment:
I suggest using `stream_zone_sync_zone` as a prefix for zones metrics. This is consistent with server zone or stream server zone metrics.
Also, the label should be `zone`, not `server_zone`.